2023-01-03_infSILAC_Processing_and_QC_of_SILAC_data.Rmd

```diff
@@ -370,13 +328,13 @@ cl_nc_correlations <- ratios_matched %>%
 #> `.groups` argument.
 
 cl_nc_correlations %>%
-  ggplot(aes(matched, cor, colour=Replicate)) +
+  ggplot(aes(matched, cor, colour=ifelse(Replicate %in% 1:8, '1:8', '9:16'))) +
```

Comment:
Colouring by the label-swap is more informative. 16 colours becomes uninterpretable, but the split into 2 colours is useful in this case.